pkg/config/config.go

```diff
@@ -737,6 +738,14 @@ type Teams struct {
 	IconURL         string `yaml:"icon_url,omitempty"`
 }
 
+type SMTP struct {
+	Enabled         bool   `yaml:"enabled,omitempty"`
+	From            string `yaml:"from,omitempty"`
+	To              string `yaml:"to,omitempty"`
```

Comment:
It would be great if we are able to send mails to multiple addresses concurrently:
```
To []string `yaml:"to,omitempty"`
```
What do you think @caarlos0?

Reply:
I think its worth it yes

Reply:
I would ask for BCC to, somtimes people reply on this kind of mails and thats quite annoying...

Reply:
fixed thank you folks